server-connect-base/Dockerfile.ubi8

```diff
@@ -66,8 +66,6 @@ enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && yum install -y confluent-schema-registry-${CONFLUENT_VERSION} \
     && echo "===> Installing Controlcenter for monitoring interceptors ..."\
     && yum install -y confluent-control-center-${CONFLUENT_VERSION} \
-    && echo "===> Installing Confluent security plugins ..." \
-    && yum install -y confluent-security-${CONFLUENT_VERSION} \
```

Comment:
@janjwerner-confluent are you saying that this command is a no-op because the package is already installed?

Reply:
That is my understanding

Reply:
For example, listing of packages installed in the latest cp-server:7.3.1

```
jwerner@C02GP0KY1PG2 ~ % docker run -it confluentinc/cp-server:7.3.1 /bin/bash
[appuser@63a3f6b9c666 ~]$ dnf list installed |grep confluent
confluent-ce-kafka-http-server.noarch 7.3.1-1                                @Confluent           
confluent-common.noarch               7.3.1-1                                @Confluent           
confluent-kafka-rest.noarch           7.3.1-1                                @Confluent           
confluent-metadata-service.noarch     7.3.1-1                                @Confluent           
confluent-rebalancer.noarch           7.3.1-1                                @Confluent           
confluent-rest-utils.noarch           7.3.1-1                                @Confluent           
confluent-security.noarch             7.3.1-1                                @Confluent           
confluent-server.noarch               1:7.3.1-1                              @Confluent           
confluent-server-rest.noarch          7.3.1-1                                @Confluent           
confluent-telemetry.noarch            1:7.3.1-1                              @Confluent    
```